### PR TITLE
Milestone 2 question history

### PR DESCRIPTION
**Pull request by Bentimate**

### CODE REVIEW

File: .vscode/settings.json
Comment (elizabethhky) [inline .vscode/settings.json:1]: please delete this file

File: assignment/assignment1/client/src/scenes/roomPage/completeQnsPopup.tsx
Comment (elizabethhky) [inline assignment/assignment1/client/src/scenes/roomPage/completeQnsPopup.tsx:1]: is this the same as the questions widget popup?

**Review by elizabethhky — CHANGES_REQUESTED** [review]
good effort! please include pictures of UI if possible :")

File: assignment/assignment1/client/src/scenes/roomPage/completeQnsPopup.tsx
Comment (Bentimate) [reply assignment/assignment1/client/src/scenes/roomPage/completeQnsPopup.tsx:1]: No, this is repurposed from the confirmationPopup

**Pull request closed without merging**